### PR TITLE
add argo files for bank2-compaction

### PR DESCRIPTION
**Pull request by cosven**
Run bank2 testcase with `compaction guard` and `gc in compaction filter` turned on. The testcase is supposed to run every 06:00 and last for 3 hours.

cc @yiwu-arbug @hicqu

### CODE REVIEW

File: config/tikv/compaction.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+force-consistency-checks = true
+
+[rocksdb.defaultcf]
+enable-compaction-guard = true
+
+[rocksdb.writecf]
+enable-compaction-guard = true
+
+[rocksdb.lockcf]
+enable-compaction-guard = true
```
Comment (mahjonp) [inline config/tikv/compaction.toml:23]: A newline is needed here.

**Review by mahjonp — APPROVED** [review]
lgtm

File: config/tikv/compaction.toml (hunk 1 above)
Comment (yiwu-arbug) [inline config/tikv/compaction.toml:23]: we don't plan to enable compaction guard for lockcf.

File: config/tikv/compaction.toml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+consistency-check-method = "mvcc"
+
+[raftstore]
+consistency-check-interval = "180s"
+
+[rocksdb]
+force-consistency-checks = true
+
+[rocksdb.defaultcf]
+enable-compaction-guard = true
```
Comment (yiwu-arbug) [inline config/tikv/compaction.toml:17]: let's also set `target-file-size-base = 128M`. This config will default to 128M when we default on compaction guard, but not right now.

File: argo/cron/bank2-compaction.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# maintainer: cosven
+
+metadata:
+  name: tipocket-bank2-compaction
```
Comment (yiwu-arbug) [inline argo/cron/bank2-compaction.yaml:4]:
```suggestion
  name: tipocket-bank2-compaction-opt
```
this is testing some compaction related optimization, not compaction itself. Also change the name of bank2-compaction.yaml.

**Review by yiwu-arbug — APPROVED** [review]
LGTM

**Pull request merged**